src/spx_resource_stats-macos.c

```diff
@@ -12,8 +12,10 @@ void spx_resource_stats_shutdown(void)
 {
 }
 
+#define TIMESPEC_TO_NS(ts) ((ts).tv_sec * 1000 * 1000 * 1000 + (ts).tv_nsec)
 
-size_t spx_resource_stats_wall_time(void)
+// Coarser (usec) wall time for macOS < Sierra
+size_t spx_resource_stats_wall_time_coarse(void)
```

Comment:
Since this function is private (not supposed to be known / called by other TUs) can you make it "static" ?